pkg/agent/openflow/framework.go

```diff
@@ -216,12 +216,10 @@ func (f *featureNetworkPolicy) getRequiredTables() []*Table {
 		if f.enableMulticast {
 			tables = append(tables,
 				MulticastEgressRuleTable,
+				MulticastEgressPodMetricTable,
 				MulticastEgressMetricTable,
 				MulticastIngressRuleTable,
 				MulticastIngressPodMetricTable,
-				MulticastEgressRuleTable,
-				MulticastEgressMetricTable,
```

Comment:
I see `MulticastEgressMetricTable` is removed because it it duplicated in the append function call. Could you explain why `MulticastEgressPodMetricTable` is removed as well?